From af9a3443822537aa259f85f09563291b394562b3 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 30 Mar 2022 07:13:54 +0530
Subject: [PATCH] Remove usage of F-string

The usage of F-string is supported only from python 3.6 and
was giving build error:
"SyntaxError: invalid syntax"

Change-Id: I3b5e9dcc4dbf12dfd1459eb7d312517058f6e228
Signed-off-by: svenate <salini.venate@intel.com>
---
 tools/releasetools/common.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index 696215e475..30a324c671 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -1661,8 +1661,7 @@ def _BuildBootableImage(image_name, sourcedir, fs_config_file, info_dict=None,
 
     BOOT_SIGNATURE_SIZE = 16 * 1024
     if len(boot_signature_bytes) > BOOT_SIGNATURE_SIZE:
-      raise ValueError(
-          f"GKI boot_signature size must be <= {BOOT_SIGNATURE_SIZE}")
+      raise ValueError('GKI boot_signature size must be <= {}'.format(BOOT_SIGNATURE_SIZE))
     boot_signature_bytes += (
         b'\0' * (BOOT_SIGNATURE_SIZE - len(boot_signature_bytes)))
     assert len(boot_signature_bytes) == BOOT_SIGNATURE_SIZE
@@ -1931,11 +1930,11 @@ def _BuildVendorBootImage(sourcedir, partition_name, info_dict=None):
   # AVB: if enabled, calculate and add hash.
   if info_dict.get("avb_enable") == "true":
     avbtool = info_dict["avb_avbtool"]
-    part_size = info_dict[f'{partition_name}_size']
+    part_size = info_dict['{}_size'.format(partition_name)]
     cmd = [avbtool, "add_hash_footer", "--image", img.name,
            "--partition_size", str(part_size), "--partition_name", partition_name]
     AppendAVBSigningArgs(cmd, partition_name)
-    args = info_dict.get(f'avb_{partition_name}_add_hash_footer_args')
+    args = info_dict.get('avb_{}_add_hash_footer_args'.format(partition_name))
     if args and args.strip():
       cmd.extend(shlex.split(args))
     RunAndCheckOutput(cmd)
-- 
2.17.1

